common/azure-common/src/main/java/com/azure/common/implementation/SwaggerMethodParser.java

```diff
@@ -1,14 +1,8 @@
-/**
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
- */
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
You are probably going to have merge errors due to changes made by Sima here - another reason to only touch what is relevant.

Reply:
Reverted